### PR TITLE
DCL/AT2: add presentation

### PR DESCRIPTION
**Pull request by tharvik**
Formatted version of the gdoc

### CODE REVIEW

File: data/DCL/projects.yaml [hunk 1]
```diff
@@ -41,6 +41,10 @@ projects:
   at2:
     name: AT2
     description: Asset Transfer System using lightweight broadcast-based primitives instead of consensus
+    layman_desc: >
+      AT2 is a full-fledged asset transfer system (cryptocurrency) without
```
Comment (ineiti) [inline data/DCL/projects.yaml:45]:
```suggestion
      AT2 is a full-fledged asset transfer system (cryptocurrency) without the need for a global
```

File: views/incubator/at2.tpl [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+process. It can handle around 7 transactions per second (TPS), which is not
+great compared to payment systems like Visa which handles 40'000 TPS.
+
+<h2>How AT2 is used in this Demo</h2>
+
+In this specific case of asset transfer, one doesn’t need a “full-blockchain”,
+only some simpler primitives. Here comes AT2, which allows for the same
+capabilities of Bitcoin while being orders of magnitudes faster, around 10'000
+TPS. By avoiding the classical proof-of-work used in other blockchains, it also
+greatly reduces the total energy consumption.
```
Comment (ineiti) [inline views/incubator/at2.tpl:18]: Plus a paragraph on the demo-to-come.

File: views/incubator/at2.tpl [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  process. It can handle around 7 transactions per second (TPS), which is not
+  great compared to payment systems like Visa which handles 40'000 TPS.
+</p>
+
+<h2>How AT2 is used in this Demo</h2>
+
+<p>
+  In this specific case of asset transfer, one doesn't need a "full-blockchain",
+  only some simpler primitives. Here comes AT2, which allows for the same
+  capabilities of Bitcoin while being orders of magnitudes faster, around 10'000
```
Comment (cgrigis) [inline views/incubator/at2.tpl:19]:
```suggestion
  capabilities as Bitcoin while being orders of magnitudes faster, around 10'000
```

File: views/incubator/at2.tpl [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  TPS. By avoiding the classical proof-of-work used in other blockchains, it
+  also greatly reduces the total energy consumption.
+</p>
+
+<p>
+  In the demonstrator, you will be able to create your own account on the AT2
+  chain, that we graciously fill up with an initial amount of asset. You can
+  then send assets to some other previously registered user. You will also be
+  performing a speedtest to measure how many transactions the system can holds.
+  The main AT2 network is actually run at EPFL by some labs and interested
```
Comment (cgrigis) [inline views/incubator/at2.tpl:29]: > The main AT2 network is actually run at EPFL by some labs and interested

Do you mean "currently" here?

File: views/incubator/at2.tpl [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  capabilities of Bitcoin while being orders of magnitudes faster, around 10'000
+  TPS. By avoiding the classical proof-of-work used in other blockchains, it
+  also greatly reduces the total energy consumption.
+</p>
+
+<p>
+  In the demonstrator, you will be able to create your own account on the AT2
+  chain, that we graciously fill up with an initial amount of asset. You can
+  then send assets to some other previously registered user. You will also be
+  performing a speedtest to measure how many transactions the system can holds.
```
Comment (cgrigis) [inline views/incubator/at2.tpl:28]:
```suggestion
  performing a speedtest to measure how many transactions the system can hold.
```

File: views/incubator/at2.tpl [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+  In this specific case of asset transfer, one doesn't need a "full-blockchain",
+  only some simpler primitives. Here comes AT2, which allows for the same
+  capabilities of Bitcoin while being orders of magnitudes faster, around 10'000
+  TPS. By avoiding the classical proof-of-work used in other blockchains, it
+  also greatly reduces the total energy consumption.
+</p>
+
+<p>
+  In the demonstrator, you will be able to create your own account on the AT2
+  chain, that we graciously fill up with an initial amount of asset. You can
```
Comment (cgrigis) [inline views/incubator/at2.tpl:26]:
```suggestion
  chain, that we graciously fill up with an initial number of assets. You can
```

File: views/incubator/at2.tpl [hunk 7]
```diff
@@ -0,0 +1,31 @@
+<h2>Introduction to Blockchains</h2>
+
+<p>
+  Blockchains are around for some years now, you've probably come across the
```
Comment (cgrigis) [inline views/incubator/at2.tpl:4]:
```suggestion
  Blockchains have been around for some years now, you've probably come across the
```

**Review by cgrigis — CHANGES_REQUESTED** [review]
Minor change suggestions

File: views/incubator/at2.tpl (hunk 4 above)
Comment (tharvik) [reply views/incubator/at2.tpl:29]: Ho, right, fixing it :)

**Comment by tharvik** [comment]
Hum, the CI fails with
```
AttributeError: 'DynamicStrictYAMLLoader' object has no attribute 'comment_handling'
../../../virtualenv/python3.6.7/lib/python3.6/site-packages/ruamel/yaml/constructor.py:1320: AttributeError
```
`ruamel.yaml` was updated today, so let's hold this PR for a day, see if any patches land.

**Comment by cgrigis** [comment]
The issue has been raised to `strictyaml` as well, but for now we can fix onto the previously known-good version of `rualmel-yaml`.
#195 should fix it

**Review by cgrigis — APPROVED** [review]

**Pull request merged**